http3/error_codes.go

```diff
@@ -6,66 +6,66 @@ import (
 	"github.com/quic-go/quic-go"
 )
 
-type errorCode quic.ApplicationErrorCode
+type ErrCode quic.ApplicationErrorCode
 
 const (
-	errorNoError              errorCode = 0x100
-	errorGeneralProtocolError errorCode = 0x101
-	errorInternalError        errorCode = 0x102
-	errorStreamCreationError  errorCode = 0x103
-	errorClosedCriticalStream errorCode = 0x104
-	errorFrameUnexpected      errorCode = 0x105
-	errorFrameError           errorCode = 0x106
-	errorExcessiveLoad        errorCode = 0x107
-	errorIDError              errorCode = 0x108
-	errorSettingsError        errorCode = 0x109
-	errorMissingSettings      errorCode = 0x10a
-	errorRequestRejected      errorCode = 0x10b
-	errorRequestCanceled      errorCode = 0x10c
-	errorRequestIncomplete    errorCode = 0x10d
-	errorMessageError         errorCode = 0x10e
-	errorConnectError         errorCode = 0x10f
-	errorVersionFallback      errorCode = 0x110
-	errorDatagramError        errorCode = 0x4a1268
+	ErrNoError              ErrCode = 0x100
+	ErrGeneralProtocolError ErrCode = 0x101
```

Comment:
The H2 package uses slightly different names, adding `Code` to the name (e.g. `ErrCodeGeneralProtocolError`). Should we do the same?

Reply:
I think (but not I've not verified) the Go team convention/idiomatic naming if that the named values of a pseudo "enum"  have to all start with the full name of the type or not. So ErrCode*. It's a good convention when using completion in an IDE for instance.

Reply:
Can you make the change then?